test/cim/iec61968/assetinfo/test_power_transformer_info.py

```diff
@@ -0,0 +1,21 @@
+#  Copyright 2021 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from test.cim.constructor_validation import ai_kwargs, pti_args, verify_power_transformer_info_constructor, verify_pti_args
+from zepben.evolve import PowerTransformerInfo, TransformerTankInfo
+
+
+def test_power_transformer_info_constructor_kwargs():
+    verify_power_transformer_info_constructor(
+        PowerTransformerInfo,
+        [TransformerTankInfo("test_transformer_end_info1"), TransformerTankInfo("test_transformer_end_info2")],
+        **ai_kwargs
+    )
+
+
+def test_power_transformer_info_constructor_args():
+    lbs = PowerTransformerInfo(*pti_args)
+    verify_pti_args(lbs)
```

Comment:
as an example of what i was describing above, move `verify_pti_args` into this file, as it's a leaf class.